ab3d2_source/c/game_preferences.c

```diff
@@ -67,6 +67,7 @@ void game_ApplyPreferences(void)
         Vid_BrightnessOffset_w  = Prefs_BrightnessOffset_RTG_w;
         Vid_GammaLevel_b        = Prefs_GammaLevel_RTG_b;
     } else {
+		Vid_DoubleHeight_b      = Prefs_PixelMode_b;
```

Comment:
Since this is now in both branches of the if/else, it might as well be extracted out and put before it.

Reply:
I will sort that out if you find no other issues after testing.

Reply:
done.